src/Storage/Helpers/InstanceHelper.cs

```diff
@@ -267,8 +264,8 @@ public static (string InstanceOwnerIdType, string InstanceOwnerIdValue) GetIdent
                 return (string.Empty, string.Empty);
             }
 
-            partyType = parts[0];
-            partyNumber = parts[1];
+            string partyType = parts[0];
+            string partyNumber = parts[1];
 
             string[] partyTypeHayStack = ["person", "organization"];
```

Comment:
Could consider defining person and organisation as enums or a least constants that are re-used

Reply:
Yea! Enum would be a good idea! 👍